### PR TITLE
Question bank is shuffled, bug with the total number of questions is fixed

### PR DESCRIPTION
**Pull request by alexrobincrabbe**
Hello There Gif

# Pull Request

❗ _**Please replace the text below the headers with your own comments.**_ ❗️

## Pull Request details:

- Closes issue: #

- INFO ABOUT THE WORK IN YOUR PULL REQUEST GOES HERE (Please be as descriptive as possible)

## Any Breaking changes:

- IF ANYTHING YOU'RE COMMITTING WOULD BREAK SOMETHING, INCLUDE HERE WHAT WOULD BREAK
- IF YOU HAVE NO BREAKING CHANGES, ENTER 'None'

## Associated Screenshots:

_( Feel free to add gifs/png screenshots of your feature in action 😊 )_

- IF YOU HAVE ANY SCREENSHOTS, INCLUDE THEM HERE.
- IF YOU HAVE NO SCREENSHOTS, ENTER 'None'

### CODE REVIEW

**Review by 2ndborn — APPROVED** [review]

File: assets/js/script.js [hunk 1]
```diff
@@ -205,18 +207,6 @@ function refresh(){
     window.location.reload("Refresh")
   }
 
-// Dark Mode function
```
Comment (kera-cudmore) [inline assets/js/script.js:208]: Was this meant to be removed?

**Review by kera-cudmore — CHANGES_REQUESTED** [review]
Was the dark mode meant to be removed?

**Comment by 2ndborn** [comment]
> Was the dark mode meant to be removed?

Alex said yes as kris requested yesterday

File: assets/js/script.js (hunk 1 above)
Comment (2ndborn) [reply assets/js/script.js:208]: yes

**Review by kera-cudmore — APPROVED** [review]

**Comment by kera-cudmore** [comment]
Alex, are you able to rename the js file on this branch and push that change, should resolve the conflict then :)

**Review by 2ndborn — APPROVED** [review]
Happy to approve

**Review by 2ndborn — APPROVED** [review]

**Comment by alexrobincrabbe** [comment]
> @alexrobincrabbe are you able to rename the js file on this branch and push that change, should resolve the conflict then :)

Have just done.

**Pull request merged**